tracker/views.py

```diff
@@ -39,11 +47,8 @@ def home(request):
         correctness=correctness,
         )
         table.save()
-    data = Task.objects.all()
-
+        return redirect('/')
 
```

Comment:
Here you can use `redirect(reverse('home'))` instead of redirect('/') like that. If one day, you decide to modify the path for this view from '/' to 'homepage'. You will not have to update the path everywhere it is used.
The name of the view can be defined in the urls.py file. for eg:
```
urlpatterns = [
    path('about/', MyView.as_view(), name='home'),
]
```